keepassxc-browser/_locales/en/messages.json

```diff
@@ -1199,4 +1207,4 @@
         "message": "Lock database",
         "description": "Lock database button title text."
     }
-}
+}
```

Comment:
What's happening here?

Reply:
White space change by my auto-formatter. I'll revert that.

Reply:
Fixed in [c1eb96b](https://github.com/keepassxreboot/keepassxc-browser/pull/1938/commits/c1eb96b4afd80aeb696b3ff68edbc9a0ed838543)